CONTRIBUTING.md

```diff
@@ -1,32 +1,167 @@
-The licenses for this project do not grant you rights to use any Microsoft names, logos, or trademarks.
-Microsoft's general trademark guidelines can be found at <http://go.microsoft.com/fwlink/?LinkID=254653>.
+- Issue/Bug: You've found a bug in a hack and want to report it, create an issue to track the bug.
+- Issue/Proposal: Used to propose a new hack. This will get the What The Hack team to engage with you and plug you into our contribution process.
+- Issue/Report: Want to share with us that you hosted a WTH event? We'd love to know how and where WTH content is being used. Give us your feedback, and let us know!
+- Issue/Request: Want help hosting a WTH event? WTH content is self-serve, but we're happy to meet with you and walk you through how to do it.
+- Issue/Suggestion: You have a suggestion on how to improve an existing hack.
 
-Privacy information can be found at <https://privacy.microsoft.com/en-us>
+### Before You File
+
+Before you file an issue, make sure you've checked the following:
+
+1. Check for existing issues
+    - Before you create a new issue, please do a search in [open issues](https://github.com/microsoft/WhatTheHack/issues) to see if the issue or feature request has already been filed.
+    - If you find your issue already exists, make relevant comments and add your [reaction](https://github.com/blog/2119-add-reaction-to-pull-requests-issues-and-comments). Use a reaction:
+        - 👍 up-vote
+        - 👎 down-vote
+1. For bugs
+    - You have as much data as possible. Let us know which hack has the bug. Is it in the Student guide? Coach guide? Is it a documentation issue? Or an issue with a provided resource file or solution file?
+1. For proposals
+    - Is this a net new hack topic, or should your contribution extend or modify and existing hack?
+    - It is okay to have more than one hack on the same technology, but the new hack should be an independent set of challenges that stand on their own.
+
+## Contributing a New Hack to What The Hack
+
+This section describes the guidelines for contributing a new hack to What The Hack. For contributors making an update to an existing hack, see the section below on [updating an existing hack](#contributing-an-update-to-an-existing-hack).
+
+In general, the WTH team prefers to collaborate with and assist contributors as they author new hacks. This makes the review process smoother when a new hack is ready to be published via a Pull Request. We offer the option of collaborating via Microsoft Teams in a "What The Hack" team that we maintain at Microsoft. 
+
+You may still choose to develop a new hack independently and submit it for review via the [PR process below](#use-work-in-progress-prs-for-early-feedback).
+
+### On-Boarding Process (Optional, but STRONGLY recommended)
+
+Once you have submitted an [Issue/Proposal](https://aka.ms/wthproposal) via Github, you can expect the following:
```

Comment:
```suggestion
Once you have submitted an [Issue/Proposal](https://github.com/microsoft/WhatTheHack/issues/new?labels=proposal&template=proposal.yml&title=%5BProposal%5D%3A+) via Github, you can expect the following:
```

Reply:
I thought about using that direct link with the querystring, but thought it seems subject to change if GH changes their API on this feature?  I do think we should avoid the aka.ms shortened link and send to the issues page where you can choose from one of the templates as I did in other spots in the doc.

Reply:
OK that works